### ISSUE TITLE
Prismlauncher: Cannot mix incompatible Qt library (6.7.2) with this library (6.8.1)

### ISSUE THREAD

**Issue by vitasomething** [void-linux/void-packages#54058]
Hei. 
After updating my system, Prismlauncher has stopped working. 
After running it from the command line to check whats wrong, the error i got was 'Cannot mix incompatible Qt library (6.7.2) with this library (6.8.1)'.

the packages i upgraded included qt6-gui, qt6-widgets and qt6-networking, though im not sure what package exactly is the cause. If i knew the environment variable for verbose debug output when running qt applications i would've checked that, but im inexperienced w/ all this and couldnt find anything on the internet.
Im not sure what to do except maybe wait till prismlauncher updates to a build compiled with a newer qt version. Once somebody better at this than me finds the specific library that breaks prismlauncher at the moment then i'd maybe bundle the older version with the prismlauncher package till then.

**Comment by classabbyamp** [comment]
this can sometimes happen with a partial upgrade of qt packages. do a full upgrade and it should resolve itself

**Comment by sburris0** [comment]
Is this solved now?

Having the same issue with 6.8.1 and 6.8.2. Unfamiliar with void's automated build system but I believe this may be because packages updates are still being built, and published one-by-one

I think this because:
- in this github repo I see no 6.8.1 packages, only 6.8.2
- in the void repo I see 6.8.1 AND 6.8.2 packages
- I just saw one single 6.8.1 package update to 6.8.2

TLDR the solution may be simply waiting a bit then updating your packages

**Comment by classabbyamp** [comment]
yes, this will be resolved when the builders clear: https://build.voidlinux.org
